tutorials/international/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: tutorial
+title: "International Flutter Apps"
+
+permalink: /tutorials/international/
+---
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* How to track the device's locale (the user's preferred language).
+* How to manage locale-specific app values.
+* How to extend the locales that Flutter supports.
+
+</div>
+
+<a name="How To Write International Flutter Applications"></a>
+## How To Write International Flutter Applications
+
+If your app might be deployed to users who speak another language then
+you'll need to "internationalize" it. That means you'll need to write
+the app in a way that makes it possible to "localize" values like text
+and layouts for each language or "locale" that the app
+supports. Flutter provides widgets and classes that help with
+internationalization and the Flutter libraries themselves are
+internationalized.
+
+### The `Locale` class and the `Localizations` Widget
+
+The [`Locale`](https://docs.flutter.io/flutter/dart-ui/Locale-class.html)
+class is used to identify the user's language. Mobile devices support
+setting the locale for all applications, usually via a system settings
+menu. Internationalized apps respond by redisplaying values that are
+locale-specific. For example if the user switches the device's locale
+from English to French then a Text widget that displayed "Hello World"
+would be rebuilt with "Bonjour le monde".
+
+The
+[`Localizations`](https://docs.flutter.io/flutter/widgets/Localizations-class.html)
+widget defines the locale for its child and the localized resources
+that the child depends on. The
+[`WidgetsApp`](https://docs.flutter.io/flutter/widgets/WidgetsApp-class.html)
+widget creates a Localizations widget and rebuilds it if the system's locale changes.
+
+You can always lookup an app's current locale with `Localizations.localeOf()`:
```

Comment:
We might consider a section title like "Determining the user's locale" because I imagine that's a common journey for the developer. Making it an explicit section title makes it easy to find.

Reply:
I changed the title of this section to: Tracking the Locale: The `Locale` class and the `Localizations` Widget